tests/tasks/test_create_user.py

```diff
@@ -0,0 +1,138 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from argilla.server.contexts import accounts
+from argilla.server.database import SessionLocal
+from argilla.server.models import User, UserRole
+from argilla.tasks.create_user import create_user
+from click.testing import CliRunner
+from sqlalchemy.orm import Session
+
+
+@pytest.fixture
+def db():
+    with SessionLocal() as db, db.begin():
+        yield db
+
+        db.query(User).delete()
+        db.commit()
+
+
```

Comment:
This is not necessary we're already "truncating" `users` table on our original `db` fixture:

```suggestion
```